### PR TITLE
Splitting schedules on the master for prod vs. qa

### PR DESCRIPTION
**Pull request by blarghmatey**

### CODE REVIEW

File: pillar/master/production_schedule.sls [hunk 1]
```diff
@@ -47,3 +21,19 @@ schedule:
     function: state.sls
     args:
       - edx.maintenance_tasks
+  scan_for_expiring_vault_leases:
+    maxrunning: 1
+    when: Monday 9:00am
+    function: vault.scan_leases
+    kwargs:
+      time_horizon: {{ ONE_WEEK }}
```
Comment (shaidar) [inline pillar/master/production_schedule.sls]: Wouldn't this be undefined?

**Comment by shaidar** [comment]
Just a nitpick but why is the order different for scanning vault leases in production and qa?

**Comment by blarghmatey** [comment]
Because that's where I copy pasted it :)

**Comment by shaidar** [comment]
👍

**Pull request merged**